app/javascript/components/my-toastify.js

```diff
@@ -0,0 +1,27 @@
+import React, {useEffect} from 'react'
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const MyComponent = ({greeting}) => {
```

Comment:
I advise you to use PropTypes to document that prop.

Reply:
very good, we will implement the PropTypes